### PR TITLE
Fixes #11963 - skip action links if task is not ready

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: app/helpers/remote_execution_helper.rb [hunk 1]
```diff
@@ -74,6 +74,15 @@ def template_invocation_status(task)
     end
   end
 
+  def template_invocation_actions(task, host)
+    if task.nil?
+      []
+    else
+      [display_link_if_authorized(_("Details"), hash_for_template_invocation_path(:id => task).merge(:auth_object => host, :permission => :view_foreman_tasks))]
+    end
+
```
Comment (iNecas) [inline app/helpers/remote_execution_helper.rb]: It seems even the Ruby Police is corrupt: how coule Rubocop let you go trying to smuggle this empty line?

**Comment by iNecas** [comment]
I can confirm, without the patch, it's broken, with the patch it's fixed. ACK (except the little whitespace thing)

**Comment by ares** [comment]
[test]

**Comment by ares** [comment]
Merging, thanks for review.

**Pull request merged**